src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsInterstitialObject.java

```diff
@@ -123,7 +102,7 @@ public void closeInterstitial() throws InterruptedException {
   }
 
   public void verifyInterstitialIsClosed() {
-    waitForElementNotVisibleByElement(interstitialAdWrapper);
+    waitForElementNotVisibleByElement(interstitialAd);
   }
 
   private boolean isSizeCorrect(int actualSize, int expectedSize) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "isSizeCorrect" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)